addons/medical_gui/initSettings.sqf

```diff
@@ -51,3 +51,12 @@
     [[0, 1, 2], [ELSTRING(common,Disabled), ELSTRING(Medical_Treatment,Anyone), ELSTRING(Medical_Treatment,Medics)], 1],
     false
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(showDamageToolTip),
```

Comment:
Also nickpicky, but this isn't really a tooltip is it? Should it be entry instead?

Reply:
sure, why the hell not